NDB.Covid19/NDB.Covid19.Droid/Resources/drawable/ic_bubble.xml

```diff
@@ -6,7 +6,7 @@
 	   android:insetBottom="40dp">
 	<shape xmlns:android="http://schemas.android.com/apk/res/android">
 		<solid android:color="@android:color/white"/>
-		<corners android:radius="12dp"/>
+		<corners android:radius="18dp"/>
 		<padding android:left="0dp" android:top="0dp" android:right="0dp" android:bottom="0dp" />
```

Comment:
not seeing any references to this, any specific reason for this change?

Reply:
it is used in bubble_layout.xml that is used in DisplayBubbleDialog

Reply:
No, I forgot to change it back.